i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+from typing import Callable
+import torch
+from torch import nn
+
+
+class ConformerPositionwiseFeedForward(nn.Module):
+    """
+    Conformer feedforward module
+    """
+    
+    def __init__(
+        self,
+        input_dim: int,
+        hidden_dim: int,
+        dropout: float = 0.1,
+        activation: Callable[[torch.Tensor], torch.Tensor] = nn.SiLU,
```

Comment:
Is `SiLU` the same default as used in the paper? Or is this already an improvement everyone is using?

Reply:
Yes, the nn.SiLU activation is swish activation. See [here](https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html)

Reply:
Ah, I did not know that, thanks!